editor.planx.uk/src/pages/FlowEditor/components/Flow/FeedbackPage.tsx

```diff
@@ -46,20 +90,216 @@ const getDetailedFeedback = async (feedbackId: number) => {
     query: GET_FEEDBACK_BY_ID_QUERY,
     variables: { feedbackId },
   });
-  console.log(detailedFeedback);
+
+  const combinedHelpText = `${detailedFeedback.helpText || ""} ${
+    detailedFeedback.helpDefinition || ""
+  } ${detailedFeedback.helpSources || ""}`.trim();
```

Comment:
nit: maybe this is a bit more readable for concatenating strings & filtering out any null ones? 
```
[detailedFeedback.helpText, detailedFeedback.helpDefinition, detailedFeedback.helpSources].filter(Boolean).join(" ").trim()
```

Reply:
Thanks, I wasn't aware of the simplicity of `.filter(Boolean)`